google-cloud-bigquery/lib/google/cloud/bigquery/dataset.rb

```diff
@@ -489,13 +490,17 @@ def delete force: nil
         #
         # @!group Table
         #
-        def create_table table_id, name: nil, description: nil
+        def create_table table_id, name: nil, description: nil, encryption_configuration: nil
```

Comment:
@tswast I found an example for this in line 572.
This can be broken up as
def create_table table_id, name: nil, description: nil,
                           encryption_configuration: nil